### PR TITLE
Update Storage _index.md for the table: ST1 Script available.

### PR DESCRIPTION
**Pull request by pinyuko**
I'm Penny Ko in MSFT.
There are ARG query listed in ST1,5,6,7,8 but in the table it shows "No". Thus, submitting a PR to update it.

# Overview/Summary

There is ARG query listed in ST1,5,6,7,8 but in the table it shows "No". Thus, submitting a PR to update it.

## Related Issues/Work Items

Replace this with a list of related GitHub Issues and/or ADO Work Items (Internal Only)

- To associate a GitHub Issue, use a [key word](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) preceded with the GitHub issue number.
- To associate an ADO Work Item, use the key word `AB#` succeeded with the [ADO Work Item ID](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## This PR fixes/adds/changes/removes

1. Change from "No" to "Yes" in Column "ARG/Script Available" for Row 1,5,6,7,8

## As part of this Pull Request I have

- [v] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [v] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [?] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [v] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [v] Ensured PR tests are passing
- [v] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries and/or scripts
- [?] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: docs/content/services/storage/storage-Account/_index.md [hunk 1]
```diff
@@ -16,14 +16,14 @@ The below table shows the list of resiliency recommendations for Storage Account
 {{< table style="table-striped" >}}
 | Recommendation                                                                                                                                                                                |     Category      | Impact |  State  | ARG/Script Available |
```
Comment (oZakari) [inline docs/content/services/storage/storage-Account/_index.md:17]:
```suggestion
| Recommendation                                                                                                                                                                                |     Category      | Impact |  State  | ARG Query Available |
```

File: docs/content/services/storage/storage-Account/_index.md [hunk 2]
```diff
@@ -16,14 +16,14 @@ The below table shows the list of resiliency recommendations for Storage Account
...
+| [ST-5 - Enable soft delete for recovery of data](#st-5---enable-soft-delete-for-recovery-of-data)                                                                                             | Disaster Recovery | Medium | Preview |          Yes         |
+| [ST-6 - Enable version for accidental modification and keep the number of versions below 1000](#st-6---enable-version-for-accidental-modification-and-keep-the-number-of-versions-below-1000) | Disaster Recovery | Medium | Preview |          Yes         |
+| [ST-7 - Enable point and time restore for containers for recovery](#st-7---enable-point-and-time-restore-for-containers-for-recovery)                                                         | Disaster Recovery |  Low   | Preview |          Yes         |
+| [ST-8 - Configure Diagnostic Settings for all storage accounts](#st-8---configure-diagnostic-settings-for-all-storage-accounts)                                                               |    Monitoring     |  Low   | Preview |          Yes         |
```
Comment (oZakari) [inline docs/content/services/storage/storage-Account/_index.md:26]:
```suggestion
| [ST-5 - Enable soft delete for recovery of data](#st-5---enable-soft-delete-for-recovery-of-data)                                                                                             | Disaster Recovery | Medium | Preview |          No        |
| [ST-6 - Enable version for accidental modification and keep the number of versions below 1000](#st-6---enable-version-for-accidental-modification-and-keep-the-number-of-versions-below-1000) | Disaster Recovery | Medium | Preview |          No         |
| [ST-7 - Enable point and time restore for containers for recovery](#st-7---enable-point-and-time-restore-for-containers-for-recovery)                                                         | Disaster Recovery |  Low   | Preview |          No         |
| [ST-8 - Configure Diagnostic Settings for all storage accounts](#st-8---configure-diagnostic-settings-for-all-storage-accounts)                                                               |    Monitoring     |  Low   | Preview |          No         |
```

**Comment by pinyuko** [comment]
Hi @oZakari , sounds great, thank you! I've updated as your suggestion.

**Review by ejhenry — APPROVED** [review]
LGTM 👍 thanks @pinyuko!

**Review by oZakari — APPROVED** [review]

**Pull request merged**